.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Python38\\python.exe"
```

Comment:
I'll have to ask Ana to review any Python stuff, sorry

Reply:
this seems to be on  .vscode, there is no python code in this component

Reply:
Is it just a filepath/import? If that's all it is, I'm sure it will be fine.